public/styles/main.css

```diff
@@ -3,13 +3,62 @@
 }
 
 body {
+  margin: 0;
+  padding: 0;
   display: grid;
-  padding-top: 10vh;
-  align-items: flex-start;
-  justify-items: center;
   font-family: system-ui, sans-serif;
-  font-weight: 500;
-  font-size: 10rem;
+}
+
+body.flex-page {
+  display: flex;
+  flex-direction: row;
+  width: 100vw; 
```

Comment:
you probably don't need `width` as it's a block level element. It should just fill available space. Can you try removing any width 100% or 100vw that are applied to divs and check I'm right. This is somewhat important as it reduces the odds of random scrollbars appearing

Reply:
this should be updated, thanks for the flag up